makefile

```diff
@@ -15,6 +15,23 @@ include make/default_compiler_options
 MATH ?=
 include make/libraries
 
+##
+# Set default compiler options.
+##
+CFLAGS = -I . -isystem $(EIGEN) -isystem $(BOOST) -isystem $(VIENNACL)  -isystem$(CVODES)/include -Wall -DBOOST_RESULT_OF_USE_TR1 -DBOOST_NO_DECLTYPE -DBOOST_DISABLE_ASSERTS -DNO_FPRINTF_OUTPUT -pipe -fPIC -D STAN_GPU
+CFLAGS_GTEST = -DGTEST_USE_OWN_TR1_TUPLE 
+LDLIBS =
+EXE =
+WINE =
+
+##
+# Check for STAN_GPU flag
+##
+ifneq (,$(findstring STAN_GPU, $(CFLAGS)))
+    CFLAGS += -lOpenCL
```

Comment:
How does this interact with the `make/tests` conditional addition of `-lOpenCL`? Why does it need to be specified twice like this?